metadata-db-app/src/cmr/metadata_db/data/oracle/concepts/subscription.clj

```diff
@@ -28,15 +27,14 @@
   [concept]
   (let [{{:keys [subscription-name
                  subscriber-id
-                 email-address
                  collection-concept-id]} :extra-fields
          user-id :user-id
          provider-id :provider-id} concept
         [cols values] (concepts/concept->common-insert-args concept)]
     [(concat cols ["provider_id" "user_id" "subscription_name"
-                   "subscriber_id" "email_address" "collection_concept_id"])
+                   "subscriber_id" "collection_concept_id"])
      (concat values [provider-id user-id subscription-name
-                     subscriber-id email-address collection-concept-id])]))
+                     subscriber-id collection-concept-id])]))
```

Comment:
I think ignoring the email at ingest time is probably the right call, if for nothing other than simplifying the logic. There is one potential problematic scenario I see though - what if an admin subscribes a user to a collection, but maybe misspells the user's name? We don't do any username validation; we wouldn't catch that the user has no email until processing time. And at that point, how would we notify the admin that the subscription is no good?

Reply:
I suggest we add a new ticket to add validation of EDL account an ingest time, would that work?

Reply:
Sure, I fine with that.